### PR TITLE
Add publishing config

### PR DESCRIPTION
**Pull request by Maxr1998**
Adapted from Jellyfin Kotlin SDK config. Untested since secrets are missing.

### CODE REVIEW

**Comment by nielsvanvelzen** [comment]
It tries to publish for this PR so that's definitely a problem :p

**Comment by Maxr1998** [comment]
Well, I thought that we could publish all master commits since we probably won't to version tags or something. But with the hardcoded ExoPlayer version it could be a problem..

**Comment by nielsvanvelzen** [comment]
We do need releases because you can't overwrite an existing version on Maven Central (-SNAPSHOT is an exception to this).

**Comment by Maxr1998** [comment]
Yeah, I remembered that. Kinda oof. Should we append/prepend our own string to the ExoPlayer version maybe?

**Comment by nielsvanvelzen** [comment]
Most projects use the Semver approach of appending a build number. So ExoPlayer `2.15.0` would become `2.15.0+1`, `2.15.0+2` etc.
I guess using tags like we do in android/atv/sdk is the best approach here to stay consistent and make sure we use the correct names.

**Comment by Maxr1998** [comment]
I reworked it to use the SDK's system of version tags, `v1.0.0` would become `2.15.0+1.0.0` for the current ExoPlayer version.

File: exoplayer-ffmpeg-extension/build.gradle.kts [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
 group = "org.jellyfin.exoplayer"
-if (exoplayerProject.ext.has("releaseVersion")) {
-    version = exoplayerProject.ext.get("releaseVersion")!!
+
+val exoPlayerVersion = when {
+    exoplayerProject.ext.has("releaseVersion") -> exoplayerProject.ext.get("releaseVersion")!!
+    else -> "latest"
 }
+version = "$exoPlayerVersion+${createVersionSuffix()}"
```
Comment (nielsvanvelzen) [inline exoplayer-ffmpeg-extension/build.gradle.kts:14]: Snapshot releases should end with `-SNAPSHOT`, right now it uses `+SNAPSHOT`. Same for build number.

https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN401

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
If you publish locally can you confirm:

- There is a -sources.jar file
- There is a -javadoc.jar file

File: exoplayer-ffmpeg-extension/build.gradle.kts (hunk 1 above)
Comment (nielsvanvelzen) [reply exoplayer-ffmpeg-extension/build.gradle.kts:14]: (I should've looked this up sooner :p)

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**